app.js

```diff
@@ -1,3 +1,118 @@
-require('babel-register');
-require('babel-polyfill');
-require('./server/index.js');
+// @flow
+
+import express from 'express';
+import morgan from 'morgan';
+import mincer from 'mincer';
+import path from 'path';
+import Router from 'named-routes';
+import debug from 'debug';
+import favicon from 'serve-favicon';
+import cookieParser from 'cookie-parser';
+import bodyParser from 'body-parser';
+
+import home from './routes/index';
+import about from './routes/about';
+
+import sequelize from './db';
+
+const debugServer = debug('interview:server');
+const debugError = debug('interview:error');
+
+const app = express();
+
+// view engine setup
+app.set('views', path.join(__dirname, 'views'));
+app.set('view engine', 'jade');
+
+//logging setup
+app.use(morgan('dev'));
+mincer.logger.use(debug('interview:mincer'));
+
+//assets pipeline setup
+let environment = new mincer.Environment();
+environment.enable('source_maps');
+environment.enable('autoprefixer');
+environment.appendPath('public/stylesheets');
+environment.appendPath('public/javascripts');
+environment.appendPath('node_modules');
+
+app.use('/assets', mincer.createServer(environment));
+
+if (process.env.NODE_ENV === 'production') {
+  environment.cache = new mincer.FileStore(path.join(__dirname, 'cache'));
+  environment.jsCompressor = 'uglify';
+  environment.cssCompressor = 'csswring';
+  environment = environment.index;
+}
+
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({extended: false}));
+app.use(cookieParser());
+
+//db setup
+sequelize.sync({force: true}).then(() => {
+  debugServer('connected to database');
+});
+
+//routing setup
+const router = new Router();
+router.extendExpress(app);
+router.registerAppHelpers(app);
+
+home.registerRoutes(app);
+about.registerRoutes(app);
+
+
+app.post('/add-interviewer', 'add-interviewer', async (req, res) => {
+  debugServer('POST /add-interviewer', req.body);
+
+  await sequelize.models.user.createInterviewer(req.body);
+
+  res.redirect(app.namedRoutes.build('home'));
+});
+
+app.post('/add-applicant', 'add-applicant', async (req, res) => {
+  debugServer('POST /add-interviewer', req.body);
+
+  await sequelize.models.user.createApplicant(req.body);
+
+  res.redirect(app.namedRoutes.build('home'));
+});
+
+
+// catch 404 and forward to error handler
+app.use((req, res, next) => {
+  const errorInfo = {
+    error: new Error('Not found'),
+    status: 404
+  };
+  next(errorInfo);
+});
+
+// development error handler
+// will print stacktrace
+if (app.get('env') === 'development') {
```

Comment:
Anyway, as I know we dont use env "development" anywere, so may be remove this if statement and add ```if ( app.get('env') === 'production') ``` in appropriate place?